src/components/LayerUI.scss

```diff
@@ -111,3 +111,7 @@
     }
   }
 }
+
+.disable-pointerEvents {
+  pointer-events: none !important;
```

Comment:
Do we really need important here?

Reply:
Yes, due to:

![image](https://user-images.githubusercontent.com/5153846/80612537-62e3f980-8a3c-11ea-9e7c-da747fbbbffc.png)

There are multiple ways to solve. Not sure which is best.